### PR TITLE
CMake macro correction (fix #313)

### LINKED ISSUE #313: CMake plugin %build and %install
Same as in #312:
Not ```cmake .``` but ```%cmake .``` should be in %build.

It is expanded on my system like this: (```rpm --eval "%cmake ."```)
```
  CFLAGS="${CFLAGS:--O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches  -m64 -mtune=generic}" ; export CFLAGS ; 
  CXXFLAGS="${CXXFLAGS:--O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches  -m64 -mtune=generic}" ; export CXXFLAGS ; 
  FFLAGS="${FFLAGS:--O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches  -m64 -mtune=generic -I/usr/lib64/gfortran/modules}" ; export FFLAGS ; 
  FCFLAGS="${FCFLAGS:--O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches  -m64 -mtune=generic -I/usr/lib64/gfortran/modules}" ; export FCFLAGS ; 
  LDFLAGS="${LDFLAGS:--Wl,-z,relro }" ; export LDFLAGS ; 
  /usr/bin/cmake \
        -DCMAKE_C_FLAGS_RELEASE:STRING="-DNDEBUG" \
        -DCMAKE_CXX_FLAGS_RELEASE:STRING="-DNDEBUG" \
        -DCMAKE_Fortran_FLAGS_RELEASE:STRING="-DNDEBUG" \
        -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
        -DCMAKE_INSTALL_PREFIX:PATH=/usr \
        -DINCLUDE_INSTALL_DIR:PATH=/usr/include \
        -DLIB_INSTALL_DIR:PATH=/usr/lib64 \
        -DSYSCONF_INSTALL_DIR:PATH=/etc \
        -DSHARE_INSTALL_PREFIX:PATH=/usr/share \
%if "lib64" == "lib64" 
        -DLIB_SUFFIX=64 \
%endif 
        -DBUILD_SHARED_LIBS:BOOL=ON
```

Make should be this:
```make  %{?_smp_mflags}```
This will expands on my system to this: ```make  -j4```

And finally ```make test``` should be:
```make test ARGS='-V %{?_smp_mflags}'```

I have got this from [here](https://fedoraproject.org/wiki/Packaging:Cmake).

### PR DESCRIPTION
**Pull request by cibo94**

### CODE REVIEW

**Comment by cibo94** [comment]
Ok now install macro is ```'make install DESTDIR="$RPM_BUILD_ROOT"'```

File: rpg/plugins/project_builder/cmake.py [hunk 1]
```diff
@@ -11,9 +11,9 @@ def patched(self, project_dir, spec, sack):
             spec.BuildRequires.add("cmake")
             logging.debug('CMakeLists.txt found')
             build = Command()
-            build.append("cmake .")
-            build.append("make")
-            install = Command("make install DESTDIR=$RPM_BUILD_ROOT")
+            build.append("%cmake .")
```
Comment (jsilhan) [inline rpg/plugins/project_builder/cmake.py]: drop this change (`%cmake` has if statement inside)

File: rpg/plugins/project_builder/cmake.py [hunk 2]
```diff
@@ -11,9 +11,9 @@ def patched(self, project_dir, spec, sack):
...
+            build.append("make %{?_smp_mflags}")
```
Comment (jsilhan) [inline rpg/plugins/project_builder/cmake.py]: use rather `%make_build` macro, please

**Comment by cibo94** [comment]
Now it is like you wanted

**Comment by jsilhan** [comment]
Merged, thanks

**Pull request closed without merging**